src/com/xilinx/rapidwright/util/Timer.java

```diff
@@ -0,0 +1,197 @@
+/*
```

Comment:
This class appears to measure execution runtime.  Perhaps we can find a more descriptive class name.